### PR TITLE
Move md to html conversion to ppo script

### PR DESCRIPTION
**Pull request by daming-lu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,10 +20,14 @@ before_install:
...
-  -  GOPATH=/tmp/go go get -u github.com/wangkuiyi/ipynb/markdown-to-ipynb
+  -  GOPATH=/tmp/go go get -u github.com/wangkuiyi/ipynb/markdown-to-ipynb          # change branch
+  -  export DEPLOY_DOCS_SH=https://raw.githubusercontent.com/PaddlePaddle/PaddlePaddle.org/move_book_conversion_logic_to_PPO_script/scripts/deploy/deploy_docs.sh
+  -  export TRAVIS_BRANCH=develop # debug purpose
 script:
   -  PATH=/tmp/go/bin:$PATH .travis/precommit.sh
-  - .travis/deploy.sh
+  # 1:CONTENT_DEC_PASSWD, 2:SOURCE_DIR, 3:VERSION
+  - cd ..
+  - curl $DEPLOY_DOCS_SH | bash -s $CONTENT_DEC_PASSWD book $TRAVIS_BRANCH
```
Comment (cs2be) [inline .travis.yml]: can we put all this in one script line?  also we need to check if this is a pull request or not.  we should only run deploy script on a merge

Comment (cs2be) [reply .travis.yml]: Also needs to check if we are in "develop" branch

**Comment by daming-lu** [comment]
screen shot 2017-10-25 at 12 20 33 pm

It is working.

**Review by cs2be — COMMENTED** [review]
LGTM

**Pull request merged**